src/modules/logger/logger.cpp

```diff
@@ -1186,13 +1187,13 @@ int Logger::create_log_dir(tm *tt)
 
 bool Logger::file_exist(const char *filename)
 {
-	struct stat buffer;
+	struct stat buffer = {};
```

Comment:
An analysis by Pavel showed that we're spending a major part of the time in PX4 on initialisers. We should not default to them unless we know they are needed.